bin/cron_script.sh

```diff
@@ -52,6 +55,7 @@ elif [ "$HOST_CLOUD" = 'azure' ]; then
 
   # Finally, inserting image_url attribute in build-azure.json
   sed -i -e '/os_type/a \         "image_url": "'"$kn_vhd_url"'", ' build-azure.json
+  echo -e "Kubenow latest stable VHD url is: $kn_vhd_url .\n"
```

Comment:
Added in order to have at least one feedback output line about changes performed by the cron_script when it comes down to Azure. Azure was the other provider failing, but I believe not for any bug reasons. Seems to me it was a one-off error.